### DISCUSSION TITLE
How to  copy relative path of the current open file to clipboard

### DISCUSSION THREAD

**Discussion by robsonpeixoto** [LazyVim/LazyVim#1904] [Q&A] [answered]
Something that I'm always needing is copy relative path of the current open file to clipboard for:
- send to a co-work
- open the terminal on the same directory of the file
- run a command line passing the path as parameter

Are there a easy way to do it?

**Comment by dpetka2001** [comment, marked as answer]
See this [thread](https://vi.stackexchange.com/questions/3686/copy-the-full-path-of-current-buffer-to-clipboard). It seems like you can assign values directly to registers, which I also learned today after googling for it.

**Reply by robsonpeixoto** [reply]
Thanks :)
